docs/docsite/rst/roadmap/ROADMAP_2_4.rst

```diff
@@ -1,224 +1,220 @@
-  - Explore JEA support (stretch)
-  - Extended become support with network/service/batch logon types
-  - Module updates
-
-    - Split "Windows" category into multiple subs
-    - Domain user/group management modules
-    - win_mapped_drive module
-    - win_hotfix
-    - win_updates rewrite to require become
-    - win_package changes required to deprecate win_msi
-    - win_copy re-write
-
-- **Cloud Provider Support**
-
-  - AWS
-
-    - Focus on pull requests for various modules
-    - Triage existing merges for modules
-    - Module work
-
-      - elb-target-groups
-      - alb*
-      - ecs
-      - Data Pipelines
-      - VPN
-      - DirectConnect
-
-  - Azure
-
-    - Expose endpoint overrides
-    - Reformat/document module output to collapse internal API structures and surface important data (eg, public IPs, NICs, data disks)
-    - Add load balancer module
-    - Add Azure Functions module
-
-  - Google Cloud Platform
-
-    - New Module: DataProc
-    - Support for Cross-Region HTTP Load Balancing
-    - New Module: GKE
-
-- **Network Roadmap**
-
-  - Removal of ``*_template`` modules
-  - Session Tracing
-  - Refactor ansible-connection to cli
-  - Module Work
-
-    - Declarative intent modules
-    - OpenVSwitch
-
-- **Contributor Quality of Life**
-
-  - All Core and Curated modules will work towards having unit testing.
-  - More bot improvements!
-  - Test Infrastructure changes
-
-    - Shippable + Bot Integration
-
-      - Provide verified test results to the bot from Shippable so the bot can comment on PRs with CI failures.
-      - Enable the bot to mark PRs with ``ci_verified`` if all CI failures are verified.
-
-    - Windows Server 2016 Integration Tests
-
-      - Restore Windows Server 2016 integration tests on Shippable.
-
-        - Originally enabled during the 2.3 release cycle, but later disabled due to intermittent WinRM issues.
-        - Depends on resolution of WinRM connection issues.
+============================
+**Target: Aug/Mid-September 2017**
+
+Administrivia and Process
+-------------------------
+- Starting with 2.4, all items that are deprecated will be removed in 4 major releases unless otherwise stated.
+  - For example: A module that is deprecated in 2.4 will be removed in 2.8
+
+Python 2.4 and 2.5 support discontinuation
+------------------------------------------
+- Ansible will not support Python 2.4 nor 2.5 on the target hosts anymore.
+  Going forward, Python 2.6+ will be required on targets, as already is the case on the controller.
+
+Python 3 and beyond
+-------------------
+- Ansible Core Engine and Core modules will be tested on Python 3
+- Communicate with Linux distros to provide Ansible running on Python 3
+- Check for Python 3 tests on core modules and create any missing
+
+Ansible-Config
+--------------
+- New yaml format for config
+- Extend the ability of the current config system by adding an ansible-config command and add the following:
+  - Dump existing config settings
```

Comment:
Blank line needed here.
In RST blank lines are needed when changing the indent level on bullet points.